.changeset/sharp-rules-compare.md

```diff
@@ -0,0 +1,5 @@
+---
+"@hashicorp/react-consent-manager": patch
```

Comment:
nit: I believe this would qualify as a minor release since it's adding functionality.